haystack/preview/components/audio/whisper_local.py

```diff
@@ -7,8 +7,8 @@
 from haystack.preview.lazy_imports import LazyImport
 
 with LazyImport(
-    "Run 'pip install transformers[torch]==4.34.1' to install torch and "
-    "'pip install --no-deps numba llvmlite 'openai-whisper>=20230918'' to install whisper."
+    "Run 'pip install transformers[torch]' to install torch and "
+    "'pip install --no-deps numba llvmlite \"openai-whisper>=20230918\"' to install whisper."
```

Comment:
There is some good news about this. An openai-whisper release last week [removed the tiktoken pin](https://github.com/openai/whisper/blob/v20231106/requirements.txt#L6). I'll make the corresponding changes in a separate PR. Should allow us to do just 'pip install \"openai-whisper>=20231106\"' here.